gradlew

```diff
@@ -1,25 +1,59 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
```

Comment:
Am I correct in understanding that this copyright (and license) apply to the contents of this file? The (c) message seems vague and meaningless. Is this supposed to be (c)  2022 SIL International? Since HTA is licensed under MIT, should that license be used here as well, instead of the Apache license?